CityOfDetroit/bloom
# fix: under construction

# Pull Request Template

## Issue Overview

This PR addresses #1564 


## Description

This PR changes all visual references of "Coming Soon"  to "Under Construction" without making any backend changed. This includes image tag labeling, homepage "under construction section", filtering, what to Expect text, partner's form questions and details, and a new display pattern to highlight when residents should apply.

Note (cc @eajensenwa):  
I made the decision to still include the "Residents should apply in..." messaging when there was only a year present but no season since I felt it was still applicable for properties that wouldn't be open for a while and didn't have a season yet set. Let me know if this is what you'd expect. 

Also, made two small assumptions on the "Residents should apply in" Figma. One that the font weight of 400 in the design differing from the Alert Box default of 500 is not worth pursuing customization. Secondly, the icon used in the Figma isn't one of our internal icons (hollow info icon is but solid is not) and with how we type the Alert Box component we would have to make a change to the component in UI-C. I assumed this change was also not worth the additional time. Let me know if you agree with these assumptions!

Note (cc @sarahlazarich ): There aren't currently any what to expect translations so I did not implement the "This property is still under construction" translations

## How Can This Be Tested/Reviewed?

One way to test this would be to pull this down locally, reseed!, and create a new listing.
1. Set it as Under Construction (ensure text update there)
2. Save
3. Verify the updates are reflected on the homepage (under construction section, see all under construction, image tag)
4. Check translations
5. Go back to partners and add marketing season and date to same listing.
6. Save
7. View listing on detail view
8. Verify new "Residents should apply in..." messaging matches Figma
9. Check translations
10. Change window sizing to ensure mobile view matches Figma

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
